testutil/ciao-down/workload.go

```diff
@@ -76,6 +81,40 @@ func (wkld *workload) save(ws *workspace) error {
 }
 
 func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
+	if err == nil {
+		switch u.Scheme {
+		case "http", "https":
+			workloadFile, err := ioutil.TempFile("/tmp", ".workload")
+			if err != nil {
+				break
+			}
+
+			absPath = workloadFile.Name()
+			defer os.Remove(absPath)
```

Comment:
Could you call defer like this

defer func() { _ = os.Remove(absPath) }()

This is the idiom we use elsewhere in the code an the reason for doing this is that it allows us to explicitly discard errors from deferred functions.  This is important if we want a clean report from errcheck.